### DISCUSSION TITLE
[Feature] "Source" menu option under LIBRARY for browsing

### DISCUSSION THREAD

**Discussion by gitterspec** [immich-app/immich#13133] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

This would allow browsing by Uploads and different External Libraries. After clicking it, we would see:

Uploads
[Ext Library Name 1]
[Ext Library Name 2]
[etc...]

I saw that browsing by folders is already implemented, but this adds new functionality because one external library could pull from many folders. This view would group all those files together under the same external library name.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3150

**Comment by gitterspec** [comment]
That feature talks about "hard splitting" libraries and doesn't mention the external libraries in particular, so I think that feature request is quite different in scope.
